tests/integration/testlib/statelib.py

```diff
@@ -77,7 +78,15 @@ def filter(self, based_on_state):
             for ifstate in self._state[INTERFACES]
             if ifstate['name'] in base_iface_names
         ]
-        self._state = {INTERFACES: filtered_iface_state}
+        for rt_type in (Route.RUNNING, Route.CONFIG):
```

Comment:
Please do not abbreviate variables if not necessary, `route_type` works fine here IMHO.

Reply:
Not sure if the name really makes sense because `type` is very generic and I would not associate this with running vs. config., I think. But I do not have a better suggestion atm.

Reply:
IMO abbreviation in small scopes (few lines) is not a problem as you can see what it is immediately.
If you extract it to its our function, it would probably solve this issue and help reading in general:
```
self._state[Route.KEY][Route.RUNNING] = _filter_routes(Route.RUNNING, ...)
self._state[Route.KEY][Route.CONFIG] = _filter_routes(Route.CONFIG, ...)
```

Reply:
Any unnecessary abbreviation steals important brain computation time that is not available for other tasks whenever one is reading the code. The only gain is saving a few characters to type...

Reply:
I don't want to join this fight, but the linux kernel code style said:

```
LOCAL variable names should be short, and to the point.
```

That's why I am use the `rt_type` initially. Anyway, I will change to whatever way to get this drag-along PR merged.